src/main/java/com/github/danielflower/mavenplugins/gitlog/renderers/SimpleHtmlRenderer.java

```diff
@@ -73,11 +73,27 @@ public void renderCommit(RevCommit commit) throws IOException {
 
 		tableHtml.append("\t\t<tr>")
 				.append("<td class=\"date\">").append(date).append("</td>")
-				.append("<td>").append(message).append("</td>")
 				.append("<td>").append(authorHtml).append("</td>")
+				.append("<td>").append(message).append("</td>")
 				.append("</tr>").append(NEW_LINE);
 	}
 
+	private String formatLongMessage(String gitMessage) {
+		String lines[] = StringEscapeUtils.escapeHtml4(gitMessage).split(LINE_SEPARATOR);
+		if (lines.length <=1) {
+			return gitMessage;
+		}
+		StringBuilder builder = new StringBuilder();
+		for (int i = 0; i < lines.length; i++) {
```

Comment:
I'm not sure if I get you. I want to emphasize first line of commit message (title), hence the if else part. First line does not have the 'commitDetails' class. That's how it looks like in the source:
``` html
<td>Enable to filter commits by a commiter name<br><span class="commitDetails"><br>Rationale: maven-jgitflow-plugin in contrary to maven-release-plugin does not provide any marker in the commit message.<br>It's not uncommon to have an artificial user handle project building and releasing (i.e. 'jenkins'). With this change, I can easily<br>filter commit messages by user name.<br></span></td>
```